src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/OrganisationStatusViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway
+{
+    public class OrganisationStatusViewModel : RoatpGatewayPageViewModel
+    {
+        public DateTime? ApplicationSubmittedOn { get; set; }
+        public DateTime SourcesCheckedOn { get; set; }
+
+        public string UkrlpData { get; set; }
```

Comment:
A 'status' suffix might be more descriptive for these properties

Reply:
Renaming is done